shell/platform/windows/flutter_windows_view.h

```diff
@@ -206,6 +208,16 @@ class FlutterWindowsView : public WindowBindingHandlerDelegate {
 
   // Currently configured WindowBindingHandler for view.
   std::unique_ptr<flutter::WindowBindingHandler> binding_handler_;
+
+  // target size during resize event
+  size_t pending_width_, pending_height_;
```

Comment:
Call these `resize_target_width_` and `resize_target_height_`. Also expand the comment to write a note that these are only valid then `resize_pending_` is set to `true`.